### PR TITLE
[build]: Bumping vllm version to v0.5.3.post1

### PR DESCRIPTION
**Pull request by oandreeva-nv**
#### What does the PR do?
Bumping vllm version to the latest.

#### Checklist
- [X] PR title reflects the change and is of format `<commit_type>: <Title>`
- [X] Changes are described in the pull request.
- [ ] Related issues are referenced.
- [X] Populated [github labels](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels) field
- [X] Added [test plan](#test-plan) and verified test passes.
- [X] Verified that the PR passes existing CI.
- [X] Verified copyright is correct on all changed files.
- [X] Added _succinct_ git squash message before merging [ref](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
- [X] All template sections are filled out.
- [ ] Optional: Additional screenshots for behavior/output changes with before/after.

#### Commit Type:
Check the [conventional commit type](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#type)
box here and add the label to the github PR.
- [X] build
- [ ] ci
- [ ] docs
- [ ] feat
- [ ] fix
- [ ] perf
- [ ] refactor
- [ ] revert
- [ ] style
- [ ] test

#### Related PRs:

vllm_backend : https://github.com/triton-inference-server/vllm_backend/pull/47

#### Test plan:
- CI Pipeline ID: 16687396

#### Related Issues: (use one of the action keywords Closes / Fixes / Resolves / Relates to)
- closes GitHub issue: #xxx

### CODE REVIEW

**Review by rmccorm4 — DISMISSED** [review]

File: build.py [hunk 1]
```diff
@@ -76,7 +76,7 @@
         "2024.0.0",  # ORT OpenVINO
         "2024.0.0",  # Standalone OpenVINO
         "3.2.6",  # DCGM version
-        "0.5.0.post1",  # vLLM version
+        "v0.5.3.post1",  # vLLM version
```
Comment (rmccorm4) [inline build.py:79]: Should the `v` be included?

Comment (oandreeva-nv) [reply build.py:79]: good catch, I'll remove it for consistency with previous specs. CI passed though with `pip install vllm==v0.5.3.post1` though

Comment (oandreeva-nv) [reply build.py]: adjusted

Comment (rmccorm4) [reply build.py]: Oh nice, I didn't know it would work both ways, just thought it was a typo. No preference either way if it works.

**Review by fpetrini15 — APPROVED** [review]

**Review by rmccorm4 — APPROVED** [review]

**Pull request merged**